src/dbnode/generated/proto/snapshot/snapshot_metadata.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package snapshot;
+
+message Metadata {
+  int64 snapshotIndex = 1;
+  bytes snapshotUUID = 2;
+  bytes commitlogIdentifier = 3;
```

Comment:
nit: Can we use commitLogID vs commitLogIdentifier?  I just tend to see more usage of ID than Identifier in our codebase already.